packages/iTwinUI-react/src/core/ComboBox/ComboBox.tsx

```diff
@@ -180,18 +233,45 @@ export const ComboBox = <T,>(props: ComboBoxProps<T>) => {
     });
   }
 
+  const getSelectedIndexes = () => {
+    const indexArray: number[] = [];
+    isMultipleEnabled(valueProp, multiple) &&
```

Comment:
why not just an `if` condition? 😄

Reply:
I wanted to make things more complex 😃 
Added if condition.